x/distribution/types/expected_keepers.go

```diff
@@ -1,21 +1,64 @@
 package types
 
-import sdk "github.com/cosmos/cosmos-sdk/types"
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+)
 
-// expected staking keeper
+// StakingKeeper expected staking keeper
 type StakingKeeper interface {
+	// iterate through validators by operator address, execute func for each validator
+	IterateValidators(sdk.Context,
+		func(index int64, validator staking.ValidatorInterface) (stop bool))
```

Comment:
Instead of requiring access to `staking` which is what we're trying to accomplish here, the `staking.ValidatorI` can be defined here.

Reply:
see previous comment. I'll check if it uses all the functions from the interface, if it does then I'd not duplicate code and import it directly from `staking`